indico/legacy/webinterface/tpls/RoomBookingNewBookingInfoWidget.tpl

```diff
@@ -1,44 +1,52 @@
 <%page args="form=None"/>
 
 <%
-    fields = ['booked_for_user', 'contact_email', 'contact_phone', 'booking_reason']
+    fields = ['booked_for_user', 'booking_reason']
     field_args = {
         'booking_reason': {'rows': 3, 'placeholder': _(u'Reason...')}
     }
     helpers = {
-        'contact_email': 'You can specify multiple email addresses separated by commas.',
-        'booking_reason': 'Specify why you need this room.'
+        'booking_reason': 'Specify why you need this room.',
+        'booked_for_user': 'Specify who will be using the room.'
+    }
+    ids = {
+        'booked_for_user': 'booked-for-user-wrapper',
+        'booking_reason': 'booking-reason'
     }
 %>
 
 <div id="bookedForInfo">
+    <div id="other-user">
+        <input type="radio" name="other_user" value="False">${ _("I'll be using the room") }<br>
+        <input type="radio" name="other_user" value="True">${ _("I'm making the booking for someone else") }<br>
+    </div>
     % for field in fields:
-        <div class="toolbar thin space-before space-after">
+        <div id="${ ids[field] }" class="toolbar thin space-before space-after">
             <div class="group">
-                % if not form[field].short_name == 'booking_reason':
-                    <span class="i-button label heavy">
-                        ${ form[field].label.text }
-                    </span>
-                % endif
                 ${ form[field](**field_args.get(field, {})) }
                 % if helpers.get(field):
                     <i class="info-helper" title="${ helpers[field] }"></i>
                 % endif
             </div>
         </div>
     % endfor
+    <input id="current-user" type="hidden" value="">
```

Comment:
Shouldn't we fill `value` with the current user's ID then, instead of using JS to do it? We could just disable the field if the other option was chosen.